nservicebus/concept-overview.md

```diff
@@ -11,53 +11,67 @@ A high level overview of the concepts, features and vernacular of NServiceBus.
 
 ### [Message](/nservicebus/messaging/)
 
-A Message is the unit of communication for NServiceBus. Types of messages:
+A Message is the unit of communication for NServiceBus. Messages are send and received by endpoints. There are two general types of messages:
 
  * [Command](/nservicebus/messaging/messages-events-commands.md): Used to request that an action should be taken.
  * [Event](/nservicebus/messaging/messages-events-commands.md): Used to communicate that some action has taken place.
 
-Message type is usually defined via marker interfaces. 
+Message type can be set either using marker interfaces `ICommand` and `IEvent` or via [conventions](/nservicebus/messaging/unobtrusive-mode.md) (so called *unobtrusive mode*).
 
+### Body
 
-### [Message Conventions](/nservicebus/messaging/messages-events-commands.md)
+The payload of the message is also called body. It travels between the endpoints in a serialized form (either textual or binary). Body of a message is not being tempered with by NServiceBus in any way.
```

Comment:
you mean tampered, right? Is that last sentence really correct, e.g. mutators? Is that important to mention here?